Makefile

```diff
@@ -41,20 +41,6 @@ build:
 install:
 	go install ./cmd/fastly
 
-LDFLAGS = -ldflags "\
- -X 'github.com/fastly/cli/pkg/version.AppVersion=${VERSION}' \
- -X 'github.com/fastly/cli/pkg/version.GitRevision=$(shell git rev-parse --short HEAD || echo unknown)' \
- -X 'github.com/fastly/cli/pkg/version.GoVersion=$(shell go version)' \
- "
-
 .PHONY: release
 release:
-	go build -o fastly $(LDFLAGS) ./cmd/fastly
-	tar czf fastly_${VERSION}_$(shell go env GOOS)-$(shell go env GOARCH).tar.gz fastly
-	rm fastly
-
-.PHONY: release-exe
-release-exe:
-	go build -o fastly.exe $(LDFLAGS) ./cmd/fastly
-	tar czf fastly_${VERSION}_$(shell go env GOOS)-$(shell go env GOARCH).tar.gz fastly.exe
-	rm fastly.exe
+	GOVERSION=$(shell go version) goreleaser
```

Comment:
I'm curious if you ran this make target, would it duplicate a tag? Once from this goreleaser process and then once from the GitHub Action which is triggered on a tag push?

Reply:
Nevermind. I think I see what's going on. The GitHub Action will run the release on `v*` tags automatically, but the make target would be for a manual release that presumably wouldn't be tagged `v*`.

Reply:
Good question, I mainly left this in for consistency with our previous setup and as you suggest to have a way to manually run the release if needed. Though I can see how it may add confusion, so may just remove.

Reply:
Have now removed :+1: